lightning/src/events/mod.rs

```diff
@@ -1404,6 +1512,21 @@ impl Writeable for Event {
 				35u8.write(writer)?;
 				// Never write ConnectionNeeded events as buffered onion messages aren't serialized.
 			},
+			#[cfg(dual_funding)]
+			&Event::OpenChannelV2Request { .. } => {
+				37u8.write(writer)?;
+				// We never write the OpenChannelV2Request events as, upon disconnection, peers
+				// drop any channels which have not yet completed any interactive funding transaction
+				// construction.
+			},
+			#[cfg(any(splicing, dual_funding))]
```

Comment:
```suggestion
	#[cfg(any(dual_funding, splicing))]
```
Let's stick to this order for consistency